### PR TITLE
320px and others sizes

### PR DESCRIPTION
**Pull request by HaniaNassalska**
I hope this is the final version. Please look at the website carefully. If you find something, write it in the comment before approval.

### CODE REVIEW

File: src/Features/Movies/PopularMovies/Core/MoviesGridWrapper/styled.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ display: grid;
         grid-gap: 16px;
         justify-items: center;
       };
-        @media(max-width: ${({ theme }) => theme.breakpoint.small}) {
```
Comment (KubaTuta) [inline src/Features/Movies/PopularMovies/Core/MoviesGridWrapper/styled.js:16]: more is better

File: src/Features/Movies/SingleMovie/Core/Credits/styled.js [hunk 2]
```diff
@@ -12,11 +12,11 @@ export const CreditWrapper = styled.div`
       @media(max-width: ${({ theme }) => theme.breakpoint.creditsMediumSmaller}) {
         grid-template-columns: repeat(3, minmax(226px, 1fr));
       };
-        @media(max-width: ${({ theme }) => theme.breakpoint.creditsSmall}) {
-          grid-template-columns: repeat(3, minmax(136px, 1fr));
+        @media(max-width: ${({ theme }) => theme.breakpoint.small}) {
+          grid-template-columns: repeat(4, minmax(136px, 1fr));
           grid-gap: 8px;
         };
-          @media(max-width: ${({ theme }) => theme.breakpoint.mobile}) {
```
Comment (KubaTuta) [inline src/Features/Movies/SingleMovie/Core/Credits/styled.js:19]: yep, to many different models on the market

File: src/Features/Movies/SingleMovie/Core/TileDetails/index.js [hunk 3]
```diff
@@ -45,30 +47,43 @@ const TileDetails = (
...
+                      ) : (
+                        ""
+                      )
+                    ) : (
+                      ""
+                    )}
+                  </Countries>
+                </Fragment>
+              ))}
+        </InfoTextWrapper>
```
Comment (KubaTuta) [inline src/Features/Movies/SingleMovie/Core/TileDetails/index.js:80]: nice logic. think cover everything

File: src/Features/People/Profile/Core/Success/MoviesGrid/styled.js [hunk 4]
```diff
@@ -3,24 +3,12 @@ import styled from "styled-components";
...
-        };
-          @media(max-width: ${({ theme }) => theme.breakpoint.mobile}) {
-            grid-template-columns: repeat(auto-fit, minmax(288px, 1fr));
-            grid-gap: 16px;
-          };
+    @media(max-width: ${({ theme }) => theme.breakpoint.small}) {
+      grid-template-columns: repeat(auto-fit, minmax(288px, 1fr));
+      grid-gap: 16px;
+      justify-items: center;
+};
```
Comment (KubaTuta) [inline src/Features/People/Profile/Core/Success/MoviesGrid/styled.js:13]: like it you make it simple

**Review by Mateusszzp — APPROVED** [review]
Everything looks good, I have no objections, I couldn't have done it better myself bravo

**Review by KubaTuta — APPROVED** [review]

**Pull request merged**